openapi/components/requestBodies/storefront/PostSubscriptionItemsChange.yaml

```
@@ -0,0 +1,92 @@
+type: object
+description: Change customer subscription items.
+properties:
+  effectiveTime:
+    type: string
+    description: Effective time when changes are applied.
+    default: now
+    enum:
+      - now
+      - next-service-period
+    x-enumDescriptions:
+      now: Apply changes immediately.
+      next-service-period: Apply changes at the beginning of next service period.
+  items:
+    type: array
+    description: Item details.
+    minItems: 1
+    items:
+      type: object
+      required:
+        - planId
+      properties:
+        planId:
+          type: string
+          description: ID of the plan.
+          maxLength: 50
+          example: plan_0YV7DENSVGDBW9S71XZNNYYQ0X
+        quantity:
+          description: Number of units of the product on the given plan.
+          type: integer
+          minimum: 0
+        usageLimits:
+          type: ['object', 'null']
+          description: |-
+            Specifies the soft and hard usage limit thresholds for an item with a metered billing plan.
+            This value is ignored when the plan is not metered.
+          properties:
+            alerts:
+              type: boolean
+              description: Specify if customer must receive spending limit alerts.
```

nitpick: [boolean fields](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#boolean-fields)
```suggestion
              description: Specifies if customer must receive spending limit alerts.
```